server/app.ts

```diff
@@ -91,6 +91,7 @@ export default function createApp(
   if (config.featureFlags.lsjOneTimeCodeAuthEnabled) {
     app.use('/oneTimeCode', requestOneTimeCodeAuthorised())
     app.use('/oneTimeCode', setUpOneTimeCode(app, oneTimeCodeService))
+    app.use('/link/request-link', (req, res) => res.redirect('/oneTimeCode/request-code'))
```

Comment:
If the feature flag is enabled and we are using one time codes rather than magic links, forward requests from the old magic link page to the new OTC page, in case any users have the old URL bookmarked etc